unic/normal/src/decompose.rs

```diff
@@ -13,17 +13,18 @@
 use std::fmt::{self, Write};
 
 use unic_ucd_normal::{CanonicalCombiningClass, decompose_canonical, decompose_compatible};
+use unic_ucd_normal::canonical_combining_class::values as ccc;
 
 
 // Helper functions used for Unicode normalization
-fn canonical_sort(comb: &mut [(char, u8)]) {
+fn canonical_sort(comb: &mut [(char, CanonicalCombiningClass)]) {
     let len = comb.len();
     for i in 0..len {
         let mut swapped = false;
         for j in 1..len - i {
             let class_a = comb[j - 1].1;
             let class_b = comb[j].1;
-            if class_a != 0 && class_b != 0 && class_a > class_b {
+            if class_a != ccc::NotReordered && class_b != ccc::NotReordered && class_a > class_b {
```

Comment:
What do you think about adding a helper method `is_reordered()`, which is `ccc != NotReordered`, and opposite of `is_not_reordered()`, to use that in these two cases?

Rational: asking people to do `!ccc.is_not_reordered()` sounds bad, and it's already used in the code, and just makes sense to have it.